<launch>
  <arg name="config_file" default="$(find aaf_walking_group)/etc/walking_groups.yaml"/>
  
  <node pkg="aaf_walking_group" type="start_walking_group.py" name="walking_group" output="screen" respawn="true">
    <param name="config_file" type="string" value="$(arg config_file)"/>
  </node>

</launch>
